pkg/flow/internal/controller/component.go

```diff
@@ -151,6 +153,12 @@ func NewComponentNode(globals ComponentGlobals, b *ast.BlockStmt) *ComponentNode
 }
 
 func getManagedOptions(globals ComponentGlobals, cn *ComponentNode) component.Options {
+	// Make sure the prefix is always absolute.
+	prefix := globals.HTTPPathPrefix
+	if !strings.HasPrefix(prefix, "/") {
```

Comment:
This is only useful in case someone using a controller manually overwrites this, right?

Reply:
Either that, or if it's set to be the empty string.